app/src/main/java/org/thoughtcrime/securesms/reactions/any/ReactWithAnyEmojiDialogFragment.java

```diff
@@ -148,18 +150,18 @@ public void onDismiss(@NonNull DialogInterface dialog) {
     callback.onReactWithAnyEmojiDialogDismissed();
   }
 
-  private void initializeViewModel() {
-    Bundle                             args       = requireArguments();
-    final MessageId messageId = new MessageId(args.getLong(ARG_MESSAGE_ID), args.getBoolean(ARG_IS_MMS));
-    viewModel = new ViewModelProvider(
-            getViewModelStore(),
-            getDefaultViewModelProviderFactory(),
-            HiltViewModelExtensions.withCreationCallback(
-                    getDefaultViewModelCreationExtras(),
-                    (ReactWithAnyEmojiViewModel.Factory factory) -> factory.create(messageId)
-            )
-    ).get(ReactWithAnyEmojiViewModel.class);
-  }
+    private void initializeViewModel() {
+        Bundle args = requireArguments();
+        final MessageId messageId = new MessageId(args.getLong(ARG_MESSAGE_ID), args.getBoolean(ARG_IS_MMS));
+        viewModel = new ViewModelProvider(
+                getViewModelStore(),
+                getDefaultViewModelProviderFactory(),
+                HiltViewModelExtensions.withCreationCallback(
+                        getDefaultViewModelCreationExtras(),
+                        (ReactWithAnyEmojiViewModel.Factory factory) -> factory.create(messageId)
+                )
+        ).get(ReactWithAnyEmojiViewModel.class);
+    }
```

Comment:
Just a minor thing but it seems this entire function has been indented two spaces more than the other functions in the file